.changeset/small-avocados-complain.md

```diff
@@ -0,0 +1,10 @@
+---
+"@khanacademy/wonder-blocks-clickable": patch
+"@khanacademy/wonder-blocks-dropdown": patch
+"@khanacademy/wonder-blocks-cell": patch
```

Comment:
**suggestion:** One more thing that I missed. `wb-cell` should be `minor` here as you are adding support to a new prop.

Reply:
Good catch - Updated! I also added more details to the changeset!